File: map/Lab7Map/Lab7Map/Domain/Nota.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab7Map.Domain
{
    public class Nota : IHasId<String>
    {
        public string id { get; set; }
        public string IdStudent { get; set; }
        public int IdTema { get; set; }
        public double  Valoare { get; set; }
        public int SaptPredare { get; set; }

        public Nota(string idStudent, int idTema, double valoare, int saptPredare)
        {
            id = idStudent + idTema;
            IdStudent = idStudent;
            IdTema = idTema;
            Valoare = valoare;
            SaptPredare = saptPredare;
        }

//        public Nota()
//        {
//            id = default(string);
//            IdStudent = default(string);
//            IdTema = default(int);
//            Valoare = default(double);
//            SaptPredare = default(int);
//        }

        public override string ToString()
        {
            return id + ";" + Valoare + ";" + SaptPredare + ";";
        }
    }
}

File: map/123/src/main/java/Models/Nota.java
package Models;

public class Nota implements HasID<String> {

    private String id;
    private String idStudent;
    private int idTema;
    private double valoare;
    private int saptPredare;

    public Nota(String idStudent, int idTema, double valoare, int saptPredare) {

        this.id = idStudent + idTema;
        this.idStudent = idStudent;
        this.idTema = idTema;
        this.valoare = valoare;
        this.saptPredare = saptPredare;
    }

    @Override
    public String getId() {
        return this.id;
    }

    @Override
    public void setId(String s) {
        id = s;
    }

    public String getIdStudent() {
        return idStudent;
    }

    public void setIdStudent(String idStudent) {
        this.idStudent = idStudent;
    }

    public int getIdTema() {
        return idTema;
    }

    public void setIdTema(int idTema) {
        this.idTema = idTema;
    }

    public double getValoare() {
        return valoare;
    }

    public void setValoare(double valoare) {
        this.valoare = valoare;
    }

    public int getSaptPredare() {
        return saptPredare;
    }

    public void setSaptPredare(int saptPredare) {
        this.saptPredare = saptPredare;
    }

    @Override
    public String toString() {
        return idStudent+";"+idTema+";"+valoare+";"+saptPredare+";";
    }
}
